frontend/views/containers/modals/DeleteGroup.vue

```diff
@@ -0,0 +1,114 @@
+<template lang='pug'>
+  form(
+    novalidate
+    ref='form'
+    name='formData'
+    @submit.prevent='submit'
+  )
+    modal-template(class='has-submodal-background is-centered')
+      template(slot='title') {{L('Leave a group')}}
+
+      h3
+        i18n Are you sure you want to delete this group?
+
+      p
+        i18n All messages exchanged between members will be
+        b
+          i18n  deleted permanently
+        | .
+
+      form(
+        novalidate
+        ref='form'
+        name='formData'
+        data-test='deleteGroup'
+        @submit.prevent='submit'
+      )
+        .field
+          label.label
+            i18n Type "Delete The Dreamers" below
+
+          input.input(
+            :class="{'error': $v.form.confirmation.$error}"
+            name='confirmation'
+            v-model='form.confirmation'
+            @keyup.enter='submit'
+            @input='$v.form.confirmation.$touch()'
+            placeholder=''
+            data-test='confirmation'
+          )
+
+          p.error(
+            v-show='$v.form.confirmation.$error'
+          )
+            i18n Please enter the sentence "
+            b
+              i18n Delete The Dreamers
+            | "
+            i18n to confirm that you delete the group
+
+        .buttons
+          button.is-outlined(
+            @click='close'
+          ) {{ L('Cancel') }}
+          button.is-danger(
+            @click='submit'
+            :disabled='$v.form.$invalid'
+          ) {{ L('Delete Group') }}
```

Comment:
@pieer these `{{ L('Delete Group') }}` type things can be written more clearly using `i18n` I think. Typically the `L` function is needed in JavaScript sections and in the markup for situations where for some reason `i18n` doesn't work. But usually in the markup the `i18n` tag should work.

Remember also that the `i18n` tag is designed to help you reduce the size of your markup by accepting a `tag` attribute (which is `span` by default`, but you can use it to be `tag='b'` or `tag='p'` or `tag='button'`, etc.

Can you simplify some of these localizations to take that into account?